package.json

```diff
@@ -34,6 +34,7 @@
     "class-validator": "^0.14.0",
     "compression": "^1.7.4",
     "dotenv": "^16.0.3",
+    "express": "^4.18.2",
```

Comment:
Why did we need to import express?

Reply:
Maybe it is because of eslint. In MDM project I allowed express reference in .eslintrc.js
"settings": {
    "node": {
      "allowModules": [
        "express"
      ],

Reply:
Thanks Audrius - I was going to say we could add `@types/express` as a dev dependency instead and import types from that (as mentioned [here](https://github.com/nestjs/nest/issues/12)) but @oscar-richardson-softwire we can go with Audrius' suggestion for consistency with MDM